odk1-src/form-logic.rst

```diff
@@ -316,7 +308,7 @@ Often, comparison `operators`_ are used in relevance expressions. For example:
 :tc:`${has_children} = 'yes'`
   True if the answer to :tc:`has_children` was ``yes``.
   
-Relevance expressions can also use `XPath functions`_.
+Relevance expressions can also use :doc:`XPath functions <form-functions>`.
```

Comment:
May be best to use just `functions` here. There's a mix of XPath and XForms and madeup functions, really.